src/main/java/homework_4/CustomFileReader/ExtraCustomFileReader.java

```diff
@@ -0,0 +1,90 @@
+package homework_4.CustomFileReader;
+
+import java.io.*;
+import java.nio.file.*;
+import java.util.Objects;
+
+public class ExtraCustomFileReader {
```

Comment:
what difference between CustomFileReader and ExtraCustomFileReader?

Reply:
Extra allows not to hardcode the file name, but only to specify the directory

Reply:
Nice job!